spec/index.bs

```diff
@@ -1955,11 +1955,17 @@ interface WGSLLanguageFeatures {
 
 {{GPUAdapterInfo}} exposes various identifying information about an adapter.
 
-None of the members in {{GPUAdapterInfo}} are guaranteed to be populated. It is at the user
+None of the members in {{GPUAdapterInfo}} are guaranteed to be populated (non-empty). It is at the user
 agent's discretion which values to reveal, and it is likely that on some devices none of the values
 will be populated. As such, applications **must** be able to handle any possible {{GPUAdapterInfo}} values,
 including the absence of those values.
 
+All {{GPUAdapterInfo}} objects from a given adapter **should** produce the same contents
+at any given time, regardless of which way they're accessed.
+The contents could change over time; for example, if some call to
+{{GPUAdapter/requestAdapterInfo()}} reveals more information (though it generally **shouldn't**),
+it should be visible via {{GPUAdapter/info|GPUAdapter.info}} as well.
```

Comment:
We should either remove [`SameObject`](https://webidl.spec.whatwg.org/#SameObject) from the WebIDL definition of `info` OR say the contents can't change over time.

Reply:
Re:below let's make it so it can't change over time, I think.

Converting to draft, I need to revise this but can't do it right now.

Reply:
Got it. Hopefully this will simplify this spec PR.

Reply:
This would mean that we can't have any user-facing prompt at requestAdapterInfo() time at all, so requestAdapterInfo() will never be used. Any prompt which affects the adapter info would need to be before requestAdapter() (otherwise the order of `adapter.info` and `adapter.requestAdapterInfo({ pleaseShowPrompt: true })` would mess things up)